src/item/SpawnEgg.php

```diff
@@ -34,7 +35,14 @@ abstract class SpawnEgg extends Item{
 
 	abstract protected function createEntity(World $world, Vector3 $pos, float $yaw, float $pitch) : Entity;
 
+	public function getEntityTypeId() : ?string{
```

Comment:
I don't really like this function. A SpawnEgg need to be able to create an entity, however, it may haven't an EntityTypeId ?

Reply:
Do you have a better way for the spawner to know what entity should be spawned when clicked with a spawn egg? I made it nullable for that reason